addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml

```diff
@@ -5,7 +5,16 @@
             <t t-if="messageInReplyToView">
                 <t t-if="!messageInReplyToView.messageView.message.parentMessage.isEmpty">
                     <b class="o_MessageInReplyToView_author text-muted ml-2">@<t t-esc="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
-                    <span class="o_MessageInReplyToView_body ml-1" t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody" t-on-click="messageInReplyToView.onClickReply"/>
+
+                    <t t-if="messageInReplyToView.messageView.message.parentMessage.isEmptyBody">
+                        <span class="o_MessageInReplyToView_body ml-1 font-italic" t-on-click="messageInReplyToView.onClickReply">
+                            Click to see the attachments <span class="fa fa-image"/>
+                        </span>
+                    </t>
```

Comment:
It's wrong to assume empty body necessarily means there are attachments.
For instance, could have no attachments but there is a subtype description or some tracking values.
You should make sure there are at least some attachments.